docs/hub/webhooks.md

```diff
@@ -193,3 +193,13 @@ Go in the activity tab for your webhook, there you will see the list of recent e
 You will see the HTTP status code and the payload of past events. You can replay those events by clicking on the `replay` button!
 
 When changing the target URL or secret of a webhook, replaying an event will send the payload to the updated URL.
+
+## FAQ
+
+##### Can I define webhooks on my organization vs. on my user account?
```

Comment:
I couldn't understand the question here, can you specify? e.g. Can I define webhooks on my organization?
(is it like you define webhooks defined from your org, affecting your user repositories?)

Reply:
`Can I define webhooks on my organization's settings like I can with my user's account?` does it sound good 😅